src/pages/Build/Research.vue

```diff
@@ -5,13 +5,15 @@
 			<div class="small-12 columns">
 				<div v-for="year in years" :key="year">
 					<h3>{{ year }}</h3>
+					<!-- eslint-disable vue/no-use-v-if-with-v-for -->
```

Comment:
We should come back and fix this too. It could use a computed property that returns the papers grouped by year so that the template can be a simple v-for